tests/test_annotations.py

```diff
@@ -54,18 +54,27 @@ def test_round(self):
                 ), "Numbers did not round to 3 or less decimals places"
 
     def test_group_annotations(self):
-        self.df.preprocess()
-        for column in self.df.file.columns:
+        cluster = Annotations(
+            '../tests/data/cluster_example.txt', ['text/csv', 'text/plain', 'text/tab-separated-values']
+        )
+        cluster.preprocess()
+        for column in cluster.file.columns:
             # Ensure labels are strings
             header = column[0]
             assert isinstance(header, str)
             annot_type = column[1]
             if annot_type == 'group':
                 # corrected testings of dataframe column dtype, using != always returns True
                 self.assertFalse(
-                    np.issubdtype(self.df.file[column].dtypes, np.number),
+                    np.issubdtype(cluster.file[column].dtypes, np.number),
                     "Group annotations must be string values",
                 )
+                # Each value in column should be treated as a string
+                for value in cluster.file[column]:
+                    self.assertTrue(
+                        type(value)== str,
```

Comment:
Linter seems to have missed:

```suggestion
                        type(value) == str,
```